cmd/keymasterd/idp_oidc.go

```diff
@@ -451,11 +445,20 @@ func (state *RuntimeState) idpOpenIDCTokenHandler(w http.ResponseWriter, r *http
 
 }
 
-func (state *RuntimeState) getUserAttributes(username string, attributes []string) (map[string][]string, error) {
+func (state *RuntimeState) getUserAttributes(username string,
+	attributes []string) (map[string][]string, error) {
+	if state.gitDB != nil {
```

Comment:
just like with get user groups this needs to be split and have ldap with higher priority

Reply:
Done.